src/pkg/caendr/caendr/utils/local_file.py

```diff
@@ -69,7 +84,7 @@ def create(self):
       self.file.save(self.__local_path)
     except Exception as ex:
       raise self._make_save_error() from ex
-    return self.__local_path
+    return self
```

Comment:
Return object itself when entering context.  When combined with changes that generalize filepath handling to `os.PathLike`, this means we can pass the `LocalFile` object around and use it directly